### ISSUE TITLE
Generating collection-level MARC records from EADs/Resource records

### ISSUE THREAD

**Issue by faithc** [pulibrary/pulfalight#540]
The current workflow for generating (collection-level) MARC from EADs in SVN will need to be revised so that they are created from Resource records. 

In case this information is helpful, Corey Schmidt at the University of Georgia created a Batch exporter tool for Windows users<https://github.com/uga-libraries/ASpace_Batch_Export-Cleanup-Upload> to export records from ArchivesSpace including EAD and MARCXML, among other things. Also, Kevin Schlottmann and David W. Hodges at Columbia University Libraries have created several tools for automating work in ASpace, including ILS catalog sync. More information about these resources can be found on the ASpace 2020 member forum page: https://archivesspace.atlassian.net/wiki/spaces/ADC/pages/1422819333/ArchivesSpace+Member+Forum+2020. 

Thanks.

**Comment by faithc** [comment]
Vacillating on whether this is release blocking issue...does need to happen sooner rather than later.

**Comment by tpendragon** [comment]
I'd recommend against totally new features blocking release, since they have the most surprising things that need to change with them.

**Comment by tpendragon** [comment]
@faithc @ccleeton This is a pretty big thing, a new feature, and I'm not sure we have it fully scoped out what this looks like. Can we push this out of blocking release?

**Comment by faithc** [comment]
Just so you're aware, I had actually initially requested this as an l-support ticket, per a suggestion from Systems staff, in August. It fell by the wayside there so it was recommended that I include it here. This generation currently happens now with EADs in SVN as a way to increase discoverability for researchers using the catalog. I think we can remove it as blocking release, though it's definitely a high priority issue.

**Comment by tpendragon** [comment]
@faithc I've removed it from blocking release, moved it to the backlog, and marked it as a priority for when we look at MVP tickets.

**Comment by tpendragon** [comment]
@faithc Who do you think knows the existing process best, at a technical level? @escowles ? @sdellis ? @kevinreiss ? @jpstroop ? Looking for how it works now so we can figure out an equivalent, and also wondering if this is something we should look at with an ILS migration looming.

**Comment by escowles** [comment]
@tpendragon It's @mzelesky who knows this process best — taking it off his plate and into something like the Pulfalight indexer that other people could work on more easily seems like a good idea to me.

**Comment by mzelesky** [comment]
Yes, I think it would be me and @regineheberlein. We had started reviewing this process pre-pandemic. Alma certainly makes this process a lot less complex, since the records can be loaded in a much easier way.

**Comment by tpendragon** [comment]
@mzelesky So is this a thing we can do before Alma, or with the code freezes and such do we have to wait to rebuild this synchronization?

**Comment by regineheberlein** [comment]
We're working on a local interim workflow. We'll revisit automating this after the Alma migration is complete.

**Comment by tpendragon** [comment]
Alright, I'm pulling this out of on deck and into the backlog, then. Adding "blocked" too.

**Comment by faithc** [comment]
Can we unblock this now? @tpendragon

**Comment by faithc** [comment]
Yang from cataloging reported an issue with how the subject headings are coded in MARC records created from EADs. Geographic headings should be $z in MARC, but they are in $x.

**Comment by tpendragon** [comment]
I'll mark this unblocked, but we still need details from @regineheberlein about how this used to work.

**Comment by mzelesky** [comment]
My understanding is that the old process of converting collection-level EADs to MARC21 that I worked on with @regineheberlein is no longer needed since ASpace has the ability to export MARC21, but I may be mistaken.

**Comment by tpendragon** [comment]
Maybe that's the case? Attached is an example of what ASpace outputs:

https://gist.github.com/tpendragon/4331a23afce506e0c34a62981fb7ce92

**Comment by faithc** [comment]
Wondering if there's an update on this project? As a reminder, #541 is a related request. Thanks!

**Comment by regineheberlein** [comment]
Sorry, no update yet. We'll keep you posted.

**Comment by tpendragon** [comment]
Closing this ticket - @regineheberlein is implementing this in `lib-jobs` via https://github.com/pulibrary/aspace_helpers/issues/2 and progress can be followed there.
